archive/frames/models.py

```diff
@@ -158,7 +159,10 @@ def as_dict(self, include_thumbnails=False):
 
         if self.area:
             ret_dict['area'] = json.loads(self.area.geojson)
-        ret_dict['related_frames'] = list(self.related_frames.all().values_list('id', flat=True))
+        if self.configuration_type in settings.EXCLUDE_RELATED_FRAME_TYPES:
```

Comment:
Is this because there are many related frames for BPM? Should we do this for other calibration types too like Bias/Dark/Lampflat? Is this just for efficiency of using less resources on an untargeted /frames/ query?

Reply:
Yes, exactly. I don't think there is a need for other frame types, but I've left it configurable for that purpose.